### PR TITLE
docs: update attribution for recently added libraries

### PR DESCRIPTION
**Pull request by malnick**
Thank you for sending a pull request :heart:

### Checklist

Please review the [guidelines for contributing code](../CONTRIBUTING.md#contribute-code) to this repository. And check `[x]` all the boxes that apply:

- [ ] This request **pulls a feature/bugfix branch** (right side) from my fork. *Not my master.*
- [x] This request **pulls against** `ockam-network/ockam`’s **develop branch** (left side).
- [x] Build succeeds `./build` with no linter warnings or test failures.
- [x] All code matched our code formatting [conventions](../CONTRIBUTING.md#code-format).
- [x] Commit messages match our [conventions](../CONTRIBUTING.md#commit-messages) and all commits
are [signed](#signed-commits).
- [ ] This request includes tests and documentation for all new code.

### Proposed Changes
Please describe the changes in your pull request.

If this pull request resolves an already recorded bug or a feature request, be sure to link to that issue.

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -454,3 +454,97 @@ of your accepting any such warranty or additional liability.
...
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+---
+
+
+
+This project use https://github.com/letsencrypt/pkcs11key which is licensed
```
Comment (mrinalwadhwa) [inline NOTICE]: `use` -> `uses`

**Comment by mrinalwadhwa** [comment]
@malnick I suspect testify added other indirect deps, could you confirm... if it did we should add those as well

**Comment by malnick** [comment]
I was going off these two recent changes to go.mod:
- https://github.com/ockam-network/ockam/commit/2f562f21285eb3b374864eab8fd36c7914217d29#diff-37aff102a57d3d7b797f152915a6dc16
- https://github.com/ockam-network/ockam/commit/dbaa503eccdb79d099c86ce4e3f6cc0d275e3d52#diff-37aff102a57d3d7b797f152915a6dc16

Do we want to include the library the library uses? Is our attribution only limited to what is in our go.mod?

**Comment by mrinalwadhwa** [comment]
since we're embedding the lib and its libs ... I think we inherit the obligation to give attribution.. so yes.
> Do we want to include the library the library uses?

**Comment by mrinalwadhwa** [comment]
found the commit that caused the libs I couldn't account for
https://github.com/ockam-network/ockam/commit/d7ad283ca2f96bfa73698699dd87a243bf13800d#diff-37aff102a57d3d7b797f152915a6dc16

could you look those up and add their licenses as well, please

**Comment by malnick** [comment]
@mrinalwadhwa I updated the NOTICE with the specified libraries in that last commit. Let me know if this covers the bases.

**Comment by mrinalwadhwa** [comment]
@malnick reindent the newly added licenses .. the editorconfig linter currently complains `./build lint` because `.editorconfig` file currently expects space indents and indents in multiples of 4 spaces

**Comment by malnick** [comment]
@mrinalwadhwa fixed

**Pull request merged**